iroh-cli/Cargo.toml

```diff
@@ -40,6 +40,7 @@ hex = "0.4.3"
 human-time = "0.1.6"
 indicatif = { version = "0.17", features = ["tokio"] }
 iroh = { version = "0.18.0", path = "../iroh", features = ["metrics"] }
+iroh-progress = { version = "0.18.0", path = "../iroh-progress" }
```

Comment:
why the introduction of a new crate?

Reply:
From the PR description:
> 3\. For the `local-node-discovery` example, I pulled some code from `iroh-cli` into a new `iroh-progress`, so that I could re-use some progress terminal output it in the example. `iroh-progress` might not be the right name, but what do we think of a separate crate for putting in reusable UI-ish pieces. If we aren't on board with this, I can just copy and paste the logic in for now & remove the crate.

Reply:
reading is hard..

Reply:
I am not the biggest fan of making a crate this small tbh for this, I think I would prefer copy and paste for now, given the example is not being published